components/UI/Card/components/Block/Block.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import styles from './Block.module.css';
+
+export function Block({ children }: React.PropsWithChildren) {
```

Comment:
nit-nit-nit-nit:

Box?

not to be confused with the first word in BEM :)

Reply:
Maybe Section? Like separate section of card

Reply:
yep
great